R/dl_build_src_gran.R

```diff
@@ -3,10 +3,11 @@
 #' Download and build GRAN packages w/ GRAN and CRAN deps
 #' 
 #' @param GRAN.dir local directory for GRAN built packages
-#' 
+#' @param returnPackageMods logical If \code{TRUE}, a data frame of packages that were downloaded and rebuilt is returned.  
+#' @param version character current version building sources for
 #' @import httr devtools
 #' @export
-dl_build_src <- function(GRAN.dir = './GRAN'){
+dl_build_src <- function(version,GRAN.dir = './GRAN', returnPackageMods = TRUE){
```

Comment:
No need to supply version here. source builds aren't associated with a specific R version.